compiler/llvm/clangUtil.cpp

```diff
@@ -2572,7 +2572,13 @@ clang::FunctionDecl* getFunctionDeclClang(const char* name)
 llvm::Type* getTypeLLVM(const char* name)
 {
   GenInfo* info = gGenInfo;
+#if HAVE_LLVM_VER >= 120
+  llvm::MDNode* scope = info->irBuilder->getCurrentDebugLocation().getScope();
+  llvm::Type* t = llvm::StructType::getTypeByName(scope->getContext(), name);
```

Comment:
We shouldn't assume anything about debug in this function. It should use `info->llvmContext` here and delete the above line setting `scope`.